algorithmic_efficiency/workloads/librispeech_conformer/librispeech_pytorch/workload.py

```diff
@@ -239,7 +239,7 @@ def greedy_decode(
         fin_result.shape[1], device=result.device).view(
             1, -1) < result.count_nonzero(dim=1).view(-1, 1)
     fin_result.view(-1)[idxs[mask != 0]] = result[result != blank_id]
-    padding = (fin_result == 0)
```

Comment:
are we  removing brackets everywhere as a no-op change to make syntax simpler ?

Reply:
the linter demanded this